### ISSUE TITLE
How to train

### ISSUE THREAD

**Issue by markpanizales** [xhuvom/darknetFaceID#8]
I have followed your instructions up to the training part, however whenever I execute the train command I got the following results.

`mark@mark-G11CD:/media/mark/Data_Application/darknetFaceID$ ./darknet detector train cfg/face.data cfg/face.cfg darknet19_448.conv.23
face
layer     filters    size              input                output
    0 conv     32  3 x 3 / 1   416 x 416 x   3   ->   416 x 416 x  32
    1 max          2 x 2 / 2   416 x 416 x  32   ->   208 x 208 x  32
    2 conv     64  3 x 3 / 1   208 x 208 x  32   ->   208 x 208 x  64
    3 max          2 x 2 / 2   208 x 208 x  64   ->   104 x 104 x  64
    4 conv    128  3 x 3 / 1   104 x 104 x  64   ->   104 x 104 x 128
    5 conv     64  1 x 1 / 1   104 x 104 x 128   ->   104 x 104 x  64
    6 conv    128  3 x 3 / 1   104 x 104 x  64   ->   104 x 104 x 128
    7 max          2 x 2 / 2   104 x 104 x 128   ->    52 x  52 x 128
    8 conv    256  3 x 3 / 1    52 x  52 x 128   ->    52 x  52 x 256
    9 conv    128  1 x 1 / 1    52 x  52 x 256   ->    52 x  52 x 128
   10 conv    256  3 x 3 / 1    52 x  52 x 128   ->    52 x  52 x 256
   11 max          2 x 2 / 2    52 x  52 x 256   ->    26 x  26 x 256
   12 conv    512  3 x 3 / 1    26 x  26 x 256   ->    26 x  26 x 512
   13 conv    256  1 x 1 / 1    26 x  26 x 512   ->    26 x  26 x 256
   14 conv    512  3 x 3 / 1    26 x  26 x 256   ->    26 x  26 x 512
   15 conv    256  1 x 1 / 1    26 x  26 x 512   ->    26 x  26 x 256
   16 conv    512  3 x 3 / 1    26 x  26 x 256   ->    26 x  26 x 512
   17 max          2 x 2 / 2    26 x  26 x 512   ->    13 x  13 x 512
   18 conv   1024  3 x 3 / 1    13 x  13 x 512   ->    13 x  13 x1024
   19 conv    512  1 x 1 / 1    13 x  13 x1024   ->    13 x  13 x 512
   20 conv   1024  3 x 3 / 1    13 x  13 x 512   ->    13 x  13 x1024
   21 conv    512  1 x 1 / 1    13 x  13 x1024   ->    13 x  13 x 512
   22 conv   1024  3 x 3 / 1    13 x  13 x 512   ->    13 x  13 x1024
   23 conv   1024  3 x 3 / 1    13 x  13 x1024   ->    13 x  13 x1024
   24 conv   1024  3 x 3 / 1    13 x  13 x1024   ->    13 x  13 x1024
   25 route  16
   26 reorg              / 2    26 x  26 x 512   ->    13 x  13 x2048
   27 route  26 24
   28 conv   1024  3 x 3 / 1    13 x  13 x3072   ->    13 x  13 x1024
   29 conv     30  1 x 1 / 1    13 x  13 x1024   ->    13 x  13 x  30
   30 detection
Loading weights from darknet19_448.conv.23...Done!
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
Loaded: 0.030896 seconds
Region Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.438415, Avg Recall: -nan,  count: 0
1: 9.846994, 9.846994 avg, 0.000100 rate, 0.156046 seconds, 1 images
Loaded: 0.000030 seconds
Region Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.388885, Avg Recall: -nan,  count: 0
2: 7.840993, 9.646395 avg, 0.000100 rate, 0.101668 seconds, 2 images
Loaded: 0.000020 seconds
Region Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.304338, Avg Recall: -nan,  count: 0
3: 4.401259, 9.121881 avg, 0.000100 rate, 0.101112 seconds, 3 images
Loaded: 0.000020 seconds
Region Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.214405, Avg Recall: -nan,  count: 0
4: 1.664008, 8.376094 avg, 0.000100 rate, 0.090299 seconds, 4 images
Loaded: 0.000022 seconds
Region Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.146090, Avg Recall: -nan,  count: 0
5: 0.564317, 7.594916 avg, 0.000100 rate, 0.095788 seconds, 5 images
Loaded: 0.000020 seconds
Region Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.091963, Avg Recall: -nan,  count: 0
6: 0.164433, 6.851868 avg, 0.000100 rate, 0.094488 seconds, 6 images
Loaded: 0.000020 seconds
Region Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.062839, Avg Recall: -nan,  count: 0
7: 0.074724, 6.174154 avg, 0.000100 rate, 0.089121 seconds, 7 images
Loaded: 0.000020 seconds
Region Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.041137, Avg Recall: -nan,  count: 0
8: 0.054264, 5.562165 avg, 0.000100 rate, 0.094170 seconds, 8 images
Loaded: 0.000022 seconds`

I only have 1 class which is me and below is my face.cfg file I used.

`[net]
batch=1
subdivisions=1
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
max_batches = 120000
policy=steps
steps=-1,100,80000,100000
scales=.1,10,.1,.1

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

#######

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[route]
layers=-9

[reorg]
stride=2

[route]
layers=-1,-3

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=30
activation=linear

[region]
anchors = 0.738768,0.874946, 2.42204,2.65704, 4.30971,7.04493, 10.246,4.59428, 12.6868,11.8741
bias_match=1
classes=1
coords=4
num=5
softmax=1
jitter=.2
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .6
random=0`
